syft.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: syft
+  version: 0.70.0
+  epoch: 0
+  description: CLI tool and library for generating a Software Bill of Materials from container images and filesystems
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - apk-tools
```

Comment:
What's the thinking behind this?

Reply:
Good question, I stole this from the apko sample of go/install, my mistake!